bin/install-python

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+shopt -s expand_aliases
+
+function main() {
+  install_dir="$1"
+  buildpack_dir="$2"
+
+  if [ ! -d "/tmp/python/bin" ]; then
+    python_dep_name=$(get_python_from_manifest "$buildpack_dir")
+    echo "-------> Providing Python $(echo $python_dep_name | cut -d'_' -f 2) (required by the buildpack to run in cflinuxfs4)"
+
+    if [ ! -d "$buildpack_dir/dependencies" ]; then
+      setup_online "$python_dep_name" "$install_dir" "$buildpack_dir"
+    else
+      setup_offline "$python_dep_name" "$install_dir" "$buildpack_dir"
+    fi
+  elif [ $install_dir != "/tmp/python" ]; then
+    cp -r "/tmp/python/." "$install_dir"
+  fi
+
+  export LD_LIBRARY_PATH="$install_dir/lib:/lib"
```

Comment:
Do you think this should be in the format`export LD_LIBRARY_PATH="<foo>:$LD_LIBRARY_PATH"`?
For e.g., in a composer scenario, would this overwrite the env set here? https://github.com/cloudfoundry/php-buildpack/blob/511274ec9f95ba718619e5a76bd950e02268bbe2/extensions/composer/extension.py#L370

Reply:
I haven't seen any issues with this in the integration tests, but it probably is safer to just append